feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/model/AmountFormatters.kt

```diff
@@ -0,0 +1,31 @@
+package io.novafoundation.nova.feature_wallet_api.presentation.model
+
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.AbsoluteSizeSpan
+import android.text.style.ForegroundColorSpan
+import androidx.annotation.DimenRes
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.common.utils.formatting.toAmountWithFraction
+import io.novafoundation.nova.feature_wallet_api.R
+
+fun CharSequence.formatBalanceWithFraction(resourceManager: ResourceManager, @DimenRes floatAmountSize: Int): CharSequence {
```

Comment:
For the new formatters I suggest to always create a separate class so we dont need to pass dependencies as arguments. Using top-level formatters everywhere was a mistake and we are now experiencing the consequences with amount formatting for example